nodes/villager/actions/ActionBase.gd

```diff
@@ -0,0 +1 @@
+extends Node
```

Comment:
ActionBase (and SenseBase) is here so i can do this in VIllager.gd - https://github.com/marc-hugh-tom/game-off-2020/pull/2/files#diff-72d7b3391536e7126bb2fcdfd327196ad3bba4f342457a4b91cfa22ea6878c6bR85

i can't test for Action, because Action.gd includes Villager (here https://github.com/marc-hugh-tom/game-off-2020/pull/2/files#diff-c766db3b8acfab64a054fbb4efd9696ac9b9b590e0f36e268ff940f7bd76bf6fR3) and including Action in Villager.gd could cause a cyclical reference